src/command_modules/azure-cli-role/azure/cli/command_modules/role/generated.py

```diff
@@ -6,35 +6,34 @@
 # pylint: disable=line-too-long
 from __future__ import print_function
 
-from azure.mgmt.authorization.operations import RoleAssignmentsOperations, RoleDefinitionsOperations
+from azure.mgmt.authorization.operations import RoleDefinitionsOperations
 from azure.graphrbac.operations import (ApplicationsOperations, ServicePrincipalsOperations,
                                         UsersOperations, GroupsOperations)
 from azure.cli.commands import cli_command
 from azure.cli.commands.arm import register_generic_update
 
-from .custom import (create_role_assignment, list_sps, list_users, create_user, list_groups, list_apps,
+from .custom import (create_role_assignment, list_role_assignments, delete_role_assignments,
+                     list_role_definitions, delete_role_definition,
+                     show_role_create_template, create_role_definition,
+                     list_sps, list_users, create_user, list_groups, list_apps,
                      _auth_client_factory, _graph_client_factory)
 
 factory = lambda _: _auth_client_factory().role_definitions
-cli_command('role list', RoleDefinitionsOperations.list, factory)
-cli_command('role delete', RoleDefinitionsOperations.delete, factory)
-cli_command('role show', RoleDefinitionsOperations.get, factory)
-cli_command('role show-by-id', RoleDefinitionsOperations.get_by_id, factory)
+simple_output_query = '[*].{name:properties.roleName, id:name, type:properties.type}'
```

Comment:
It might be better to enforce the header casing in the table output formatter. That ensures it won't get "missed".

---
In reply to: [73722176](https://github.com/Azure/azure-cli/pull/602#discussion_r73722176) [](ancestors = 73722176)